stories/components/form/FileSelector.stories.mdx

```diff
@@ -0,0 +1,54 @@
+import { Meta, Canvas, Story, ArgsTable, Source } from '@storybook/addon-docs';
+
+import FileSelector from '@shell/components/form/FileSelector';
+
+export const Template = (args, { argTypes }) => ({
+  components: { FileSelector },
+  props:      Object.keys(argTypes),
+  template:   '<FileSelector v-bind="$props" />',
+})
+
+<Meta title="Components/Form/FileSelector" />
+
+# File Selector
+Input elements with type ‘file’ allows the user to select files from their file system or device storage.
+
+### Description
+- Associate a label to every file input for good accessibility. Do not include a colon at the end of the label.
+- Use only to upload file.
+- Set a maximum file size in order to prevent a potential service outage.
+
+<Canvas columns={2}>
+  Default
+  <Story
+    name="FileSelector"
+    args={{
+      label: 'Choose file',
+    }}>
+    {Template.bind({})}
+  </Story>
+  Disabled
+  <Story
+    name="disabled"
+    args={{
+      label: 'Choose file',
+      disabled: true
+    }}>
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+### Import 
+
+<Source
+  language='js'
+  light
+  format={false}
+  code={`
+     import FileSelector from '@shell/components/form/FileSelector';
+  `}
+/>
+
+
+### Props table
+<ArgsTable of={FileSelector} />
```

Comment:
`mode` description / values are inconsistent with `checkbox`. I can see that they are constants `_EDIT` and `_VIEW`  it would be good docs are consistent. Also say something like `_VIEW = 'view'` and `_EDIT = 'edit`

in `Checkbox`
<img width="853" alt="image" src="https://user-images.githubusercontent.com/1387263/181809326-82e6df1f-ed2e-44f2-9a9f-7aeaf6b662f6.png">

in `FileSelector`

<img width="853" alt="image" src="https://user-images.githubusercontent.com/1387263/181809432-0ffe895f-ebf7-4c0c-9811-1cf86005144c.png">